public/data/en/fields.json

```diff
@@ -5,9 +5,9 @@
     "nvrf_id": "choice_of_party",
     "name": "Choice of party",
     "label": "Enter the full name of the political party you wish to join or leave blank.",
-    "help_text": false,
```

Comment:
I would switch this text back to "instructions" rather than "help text". Help text is more so how to format the input, whereas instructions give more context and comes from the NVRF pdf.

Reply:
Ah ok, that makes sense!